git/githistory/rewriter.go

```diff
@@ -311,9 +310,14 @@ func (r *Rewriter) rewriteTree(commitOID []byte, treeOID []byte, path string, fn
 
 	entries := make([]*odb.TreeEntry, 0, len(tree.Entries))
 	for _, entry := range tree.Entries {
-		path := filepath.Join(path, entry.Name)
+		var epath string
```

Comment:
Maybe `fullpath`? No idea if this makes it clearer but I stumbled upon `epath`. Took me a second to understand the meaning.

Reply:
Good idea, I renamed this to `fullpath` in: 6c782f60.